tests/integration/api_v3/rpc/blockchainAppsTokenMetadata.test.js

```diff
@@ -86,8 +87,8 @@ describe('get.blockchain.apps.meta.tokens', () => {
 		expect(result.meta).toMap(metaSchema);
 	});
 
-	it('returns blockchain application off-chain metadata for tokens by global chainID', async () => {
-		const response = await getBlockchainAppsTokenMetadata({ chainID: '00000001' });
+	it('returns blockchain application off-chain metadata for tokens by chainID', async () => {
```

Comment:
Please apply the same changes requested above.

Reply:
Applied.